pkg/proxy/proxy_test.go

```diff
@@ -151,4 +155,50 @@ var _ = Describe("Setup", func() {
 })
 
 // TODO(qbarrand) make the DiscoveryClient in clients.HasResource injectable, so we can mock it.
-//var _ = Describe("ClusterConfiguration", func() {})
+var _ = Describe("ClusterConfiguration", func() {
+	var (
+		ctrl           *gomock.Controller
+		proxyStruct    proxy
+		mockKubeClient *clients.MockClientsInterface
+	)
+	BeforeEach(func() {
+		ctrl = gomock.NewController(GinkgoT())
+		mockKubeClient = clients.NewMockClientsInterface(ctrl)
+		proxyStruct = proxy{
+			kubeClient: mockKubeClient,
+			log:        zap.New(zap.WriteTo(ioutil.Discard)),
+		}
+	})
+
+	It("HasResource failed", func() {
+		mockKubeClient.EXPECT().HasResource(gomock.Any()).Times(1).Return(false, fmt.Errorf("some error"))
```

Comment:
I don't think we need `.Times(1)` in those calls, as that's the default.

Reply:
i just prefer to make it explicit